### PR TITLE
[FIX] Minor NFTs bugs

### PR DESCRIPTION
**Pull request by gantunesr**
## ⚠️  Blockers 

- ~PR #3380~

# Description

This development is destined to fix:

- The lack of collection name on ERC-1155 NFTs in custom networks.
- Add new attribute row indicating the NFT standard
- The "Unidentified contract" bug by accessing a different attribute in the OpenSea response object.

Fixed Contract Name         |  ERC721 Standard property | ERC1155 Standard property 
:-------------------------:|:-------------------------:|:---------:
  |   | 

# Tasks

- [x] Add controllers update
- [x] Update tests
- [x] Update snapshots (if necessary)

# References

- [MM Controllers PR](https://github.com/MetaMask/controllers/pull/628)

### CODE REVIEW

File: app/core/Engine.js [hunk 1]
```diff
@@ -173,18 +174,22 @@ class Engine {
...
+					addTokens: tokensController.addTokens.bind(tokensController),
+					getTokensState: () => tokensController.state,
+					getTokenListState: () => tokenListController.state,
+				}),
+				new CollectibleDetectionController({
+					onCollectiblesStateChange: (listener) => collectiblesController.subscribe(listener),
+					onPreferencesStateChange: (listener) => preferencesController.subscribe(listener),
+					onNetworkStateChange: (listener) => networkController.subscribe(listener),
 					getOpenSeaApiKey: () => collectiblesController.openSeaApiKey,
 					getBalancesInSingleCall:
```
Comment (Cal-L) [inline app/core/Engine.js:190]: Same issue here: https://github.com/MetaMask/metamask-mobile/pull/3386#discussion_r747827282
But as you mentioned, covered by blocking ticket.

File: app/core/Engine.js [hunk 2]
```diff
@@ -568,7 +575,6 @@ export default {
 			AddressBookController,
 			AssetsContractController,
 			CollectiblesController,
-			AssetsDetectionController,
```
Comment (Cal-L) [inline app/core/Engine.js:571]: Same as here: https://github.com/MetaMask/metamask-mobile/pull/3386#discussion_r747836587
Again, covered by blocking ticket.

**Review by Cal-L — COMMENTED** [review]
Left same comments regarding blocking PR. Awaiting that PR to be merged.

**Review by cortisiko — APPROVED** [review]
🌮 🌮 🌮 🌮 🌮

**Review by Cal-L — APPROVED** [review]
LGTM

**Pull request merged**